packages/svelte2tsx/test/emitDts/samples/typescript-declarationMap/tsconfig.json

```diff
@@ -9,7 +9,8 @@
         "esModuleInterop": true,
         "allowJs": true,
         "checkJs": true,
-        "declarationMap": true
+        "declarationMap": true,
+        "declaration": true
```

Comment:
TypeScript complains when `declarationMap` is enabled without `declaration`, the other tsconfig with `declarationMap` enabled in language-server has it already.